libkbfs/block_journal.go

```diff
@@ -167,11 +172,13 @@ func (j *blockJournal) blocksPath() string {
 	return filepath.Join(j.dir, "blocks")
 }
 
-// TODO: Consider truncating the ID string to avoid running into path
-// limits while keeping the chance of collision negligible.
 func (j *blockJournal) blockPath(id BlockID) string {
+	// Truncate to 34 characters, which corresponds to 16 random
```

Comment:
Okay, rewrote the comment. Instead of talking about probabilities, it's conceptually simpler to talk about expected number of blocks/MDs generated before getting a collision, which is in this case 2^64. Added a link to https://en.wikipedia.org/wiki/Birthday_problem#Cast_as_a_collision_problem .